.github/README.md

```diff
@@ -1,16 +1,32 @@
-# UndertaleModTool
-*(seeing such an amazing tool...<br/>
-... fills you with DETERMINATION.)*
+<p align="center">
+  <img src="images/logo.png" alt="UndertaleModTool Logo"/>
+</p>
 
-Heya. I heard you like digging deep into Undertale data so I made a tool just for you! 
+[![Underminers Discord](https://img.shields.io/discord/566861759210586112?label=Discord&logo=discord&logoColor=white)](https://discord.gg/UMwccfs2Hb) [![GitHub](https://img.shields.io/github/license/krzys-h/UndertaleModTool?logo=github)](https://github.com/krzys-h/UndertaleModTool/blob/master/LICENSE.txt)
 
-**Downloads** are **HERE**: :point_right::point_right::point_right: https://github.com/krzys-h/UndertaleModTool/releases :point_left::point_left::point_left:
+<p align="center">(seeing such an amazing tool it fills you with DETERMINATION.)</p>
+<p align="center">Heya. I heard you like digging deep into Undertale/Deltarune or any GameMaker game data so I made a tool just for you!</p>
+<p align="center">
+  <img src="images/flowey.gif" alt="Flowey"/>
+</p>
 
-**Have questions? Want to talk?** Join us on Discord! https://discord.gg/RxXpdwJ
+# Downloads
 
-![flowey](flowey.png)
+Looking for the most stable or the latest avaliable download? Both the latest stable release and the most cutting edge version are in the table below!
+
+| Releases 	| Status 	|
+|:---:	|----------	|
+| Stable[^1][^2] 	| [![GitHub release (latest by date)](https://img.shields.io/github/downloads/krzys-h/UndertaleModTool/latest/total)](https://github.com/krzys-h/UndertaleModTool/releases/latest) |
+| Bleeding edge[^1] 	| [![Build tool NET Bundled](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml/badge.svg)](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml) |
+
+[^1]: UndertaleModTool has different builds per release. The differences are as follows:
+      - `.NET bundled` - bundles the required .NET runtime version needed to run the tool alongside it. All stable releases are .NET bundled, so you don't have to worry about installing the necessary runtime.
+      - `Single file` - the tool is only one executable, with all dependencies embedded within it. This does make your folders cleaner, however it also causes some unexpected stability issues.
+      - `Non-single File` - all dependencies are not embedded within the executable, but are now located right next to it. Choose this if you don't care about finding the right executable in-between of ~300 dll's, or if the stability issues from the `Single file` build affect you.
+[^2]: You can update to the bleeding edge releases at any time from within the settings menu of UndertaleModTool.
+
```

Comment:
Option 1 use the in-built superscript notations for footnotes:
```suggestion
| Releases 	| Status 	|
|:---:	|----------	|
| Stable¹ ² 	| [![GitHub release (latest by date)](https://img.shields.io/github/downloads/krzys-h/UndertaleModTool/latest/total)](https://github.com/krzys-h/UndertaleModTool/releases/latest) |
| Bleeding edge¹ 	| [![Build tool NET Bundled](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml/badge.svg)](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml) |

¹: UndertaleModTool has different builds per release. The differences are as follows:
      - `.NET bundled` - bundles the required .NET runtime version needed to run the tool alongside it. All stable releases are .NET bundled, so you don't have to worry about installing the necessary runtime.
      - `Single file` - the tool is only one executable, with all dependencies embedded within it. This does make your folders cleaner, however it also causes some unexpected stability issues.
      - `Non-single File` - all dependencies are not embedded within the executable, but are now located right next to it. Choose this if you don't care about finding the right executable in-between of ~300 dll's, or if the stability issues from the `Single file` build affect you.
²: You can update to the bleeding edge releases at any time from within the settings menu of UndertaleModTool.

```

Option 2 get rid of footnotes entirely:
```suggestion
| Releases 	| Status 	|
|:---:	|----------	|
| Stable 	| [![GitHub release (latest by date)](https://img.shields.io/github/downloads/krzys-h/UndertaleModTool/latest/total)](https://github.com/krzys-h/UndertaleModTool/releases/latest) |
| Bleeding edge 	| [![Build tool NET Bundled](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml/badge.svg)](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml) |

It is worth noting that UndertaleModTool has different builds per release. The differences are as follows:
      - `.NET bundled` - bundles the required .NET runtime version needed to run the tool alongside it. All stable releases are .NET bundled, so you don't have to worry about installing the necessary runtime.
      - `Single file` - the tool is only one executable, with all dependencies embedded within it. This does make your folders cleaner, however it also causes some unexpected stability issues.
      - `Non-single File` - all dependencies are not embedded within the executable, but are now located right next to it. Choose this if you don't care about finding the right executable in-between of ~300 dll's, or if the stability issues from the `Single file` build affect you.
You can also update to the bleeding edge releases at any time from within the settings menu of UndertaleModTool.

```

I personally prefer option 2 more, as it reads more like a proper text and the flow of reading is not interrupted with the footnotes.